src/_alert.css

```diff
@@ -0,0 +1,73 @@
+.fieldset
+{
+    width:100%;
+}
+
+.alert
+{
+    
+    display :inline-block;
+    width:80%;
+   
+    color:white;
+    
+}
+.alert-message
+{
+    padding-bottom: 2%;
+    padding-top: 2%;
+    padding-left: 2%;
+    padding-right: 2%;
+    font-size: 24px;
+    text-align: left;
+    
+}
+.alert-button
+{
+  
+    padding-bottom: 1%;
+    text-align: center;
+  
+    
+}
+.OK:active;
+{
+   
+    color:black;
+    width:10%;
+    height:10%;
+    background-color: white;
+    
+}
+.OK:hover{
+ 
+    transform:scale(1.03);
+    opacity:3;
+    
+}
+
+
+
+
+.danger
```

Comment:
For color use the default please,  you see the palette [colors](https://github.com/rodgeraraujo/minus.css/blob/dev/src/_color.css). Use the vars to get color dynamic, like `var(--color-secondary)`.